docs/src/main/sphinx/functions/ai.md

```diff
@@ -39,14 +38,58 @@ Configure multiple catalogs to use the same functions with different LLM
 providers. In this case, the functions must be referenced using their
 fully qualified name, rather than relying on the SQL path.
 
-## Providers
+:::{list-table} AI functions configuration properties
+:widths: 40, 60
+:header-rows: 1
+
+* - Property name
+  - Description
+* - `ai.provider`
+  - Required name of the provider. Must be `anthropic` for using the
+    [Anthropic provider](ai-anthropic) or `openai` for [OpenAI](ai-openai) or
+    [Ollama](ai-ollama).
+* - `ai.anthropic.endpoint`
+  - URL for the Anthropic API endpoint. Defaults to `https://api.anthropic.com`.
+* - `ai.anthropic.api-key`
+  - API key value for Anthropic API access. Required with `ai.provider` set to
+    `anthropic`.
+* - `ai.openai.endpoint`
+  - URL for the OpenAI API or Ollama endpoint. Defaults to
+    `https://api.openai.com`. Set to the URL endpoint for Ollama when using
+    models via Ollama and add any string for the `ai.openai.api-key`.
+* - `ai.openai.api-key`
+  - API key value for OpenAI API access. Required with `ai.provider` set to
+    `openai`. Required and ignored with Ollama use.
+* - `ai.model`
+  - Required name of the model. Valid names vary by provider. Model must be
+    suitable for textual analysis. The model is used for all functions, unless a
+    specific model is configured for 
+* - `ai.analyze-sentiment.model`
+  - Optional override to use a different model for {func}`ai_analyze_sentiment`.
+* - `ai.classify.model`
+  - Optional override to use a different model for {func}`ai_classify`.
+* - `ai.extract.model`
+  - Optional override to use a different model for {func}`ai_extract`.
+* - `ai.fix-grammar.model`
+  - Optional override to use a different model for {func}`ai_fix_grammar`.
+* - `ai.generate.model`
+  - Optional override to use a different model for {func}`ai_gen`.
+* - `ai.mask.model`
+  - Optional override to use a different model for {func}`ai_mask`.
+* - `ai.translate.model`
+  - Optional override to use a different model for {func}`ai_translate`.
 
-The AI functions invoke an external LLM, which must be configured for the
-catalog. Performance, results, and cost of all AI function invocations are
-completely dependent on the LLM provider and the model used.
-You must specify a model that is suitable for textual analysis.
+:::
+
+### Providers
+
+The AI functions invoke an external LLM. Access to the LLM API must be
+configured in the catalog. Performance, results, and cost of all AI function
```

Comment:
We might mention that costs are dependent on which AI function is used (based on the implementation prompt size), the length of the input, and the length of the output from the model. This is because model providers charge based input and output tokens.

Reply:
Added that into a models section with the separate table.